doc.go

```diff
@@ -0,0 +1,55 @@
+package doc
+
+import (
+	"io"
+	"os/exec"
+)
+
+// JobsList stores a map of the Job's containing the linux process key'd by job id (in memory DB)
+type JobsList map[string]*Job
+
+// Job maintains the exec.Cmd struct (containing the underlying os.Process) and the "owner" for authz
+type Job struct {
+	owner string
+	cmd   *exec.Cmd
+}
+
+// JobOpts wraps the options that can be passed to cgroups for the job
+// details at https://facebookmicrosites.github.io/cgroup2/docs/overview
+type JobOpts struct {
+	CpuWeight int    // `cpu.weight`
```

Comment:
```suggestion
	CPUWeight int    // `cpu.weight`
```
https://go.dev/wiki/CodeReviewComments#initialisms

Reply:
updated